feat(pipes-targets): add step functions state machine target

WtfJoke opened the pull request:
### Issue #29665

Closes #29665

### Reason for this change
Step Function target is not supported yet by pipes-targets.
<!--What is the bug or use case behind this change?-->

### Description of changes
- Added step function as a pipes target.
- I've decided to make the `invocationType` a required parameter, since this made the code clearer and make users aware of how they want the step function to be invoked. 
The [AWS::Pipes::Pipe PipeTargetStateMachineParameters](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetstatemachineparameters.html) has this as an optional parameter (defaulting to Request-Response), which can lead the user unknowingly in a broken pipe, because cdk's StateMachines [default](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_stepfunctions.StateMachine.html#statemachinetype) to Standard Workflow, which is not compatible with Request-Response Invocation Type.
- Currently there seems no way to prevent users from creating a pipe with an imported Standard StateMachine and InvocationType Request-Response as the stateMachineType cant be read (or I dont know how :D)
<!--What code changes did you make? Have you made any important design decisions?-->

### Description of how you validated changes
- Added unit tests
- Added integration test
<!--Have you added any unit tests and/or integration tests?-->

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----
I've talked with @RaphaelManke and he was fine for me opening up a PR (put him as a co-author nevertheless) :)

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

aws-cdk-automation reviewed (changes requested):
The pull request linter has failed. See the aws-cdk-automation comment below for failure reasons. If you believe this pull request should receive an exemption, please comment and provide a justification.


A comment requesting an exemption should contain the text `Exemption Request`. Additionally, if clarification is needed add `Clarification Request` to a comment.

aws-cdk-automation commented:
The pull request linter fails with the following errors:

	❌ The first word of the pull request title should not be capitalized. If the title starts with a CDK construct, it should be in backticks "``".

<b>PRs must pass status checks before we can provide a meaningful review.</b>

If you would like to request an exemption from the status checks or clarification on feedback, please leave a comment on this PR containing `Exemption Request` and/or `Clarification Request`.

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 1db8840e44555cf713207ae6db1f361b2bb8ce0a
* Result: SUCCEEDED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=85539a5d-19ea-405f-bc8f-560389f631f8%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: d35ec0cdd7bf84bf2ebf5d8a41bdcb0c2fa16321
* Result: SUCCEEDED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=575c387b-eb58-483d-904c-3f0a6b4860a6%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/index.ts:
```diff
@@ -1 +1,2 @@
 export * from './sqs';
+export * from './step-function';
```
Please make sure these are named consistently 
```suggestion
export * from './stepfunctions';
```

WtfJoke replied:
Thanks for taking a look already :)

Currently its consistent using `step-function` everywhere. Should I change it to the plural version of it?

mrgrain replied:
How do you mean "everywhere"? I was referring to the module name which is `aws-stepfunctions`.

(You might totally be right about this by the way)

WtfJoke replied:
>How do you mean "everywhere"?

I meant the filenames. 

> I was referring to the module name which is aws-stepfunctions.

Ah I think I know now what you meant. The package name is `aws-stepfunctions` and I should keep that name consistent also in this pipes-targets module, changed it to stepfunctions :)

I kept the name `StepFunctionTarget` if you think I should change that as well let me know.

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/README.md:
````diff
@@ -63,3 +68,42 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
 });
 ```
+
+### AWS Step Function
````
```suggestion
### AWS Step Functions
```

WtfJoke replied:
Good catch! Changed it

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 141dbf5f9bcf11522f52ebf32b9ee971eafee30e
* Result: SUCCEEDED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=84e75013-c47c-4c3e-b3e8-10a24c59611c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 08d94b726738c14f86930b7f6de1e6be31457d10
* Result: SUCCEEDED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=60bcaf37-8927-4401-a407-612c5a6ae80f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

WtfJoke commented:
@mrgrain just wanted to ask if there is something else I should change or does an additional person needs to review my PR (or I just need to wait a bit more)?

mrgrain commented:
> @mrgrain just wanted to ask if there is something else I should change or does an additional person needs to review my PR (or I just need to wait a bit more)?

Someone else should get around to this eventually. I'm not actually working on the CDK anymore 🙈 
But ping me if you don't get a review next week!

WtfJoke commented:
Thanks, appreciate it! Lets hope its not necessary :D

WtfJoke commented:
@mrgrain unfortunately I need to come back to your offer as the PR hasnt been merged and a week passed. So I hope pinging you helps :D Thanks!

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/index.ts:
```diff
@@ -1 +1,2 @@
 export * from './sqs';
+export * from './stepfunctions';
```
```suggestion
export * from './stepfunctions';

```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,83 @@
+  /**
+   * Specify whether to invoke the Step Functions state machine synchronously (`REQUEST_RESPONSE`) or asynchronously (`FIRE_AND_FORGET`).
+   *
+   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetsqsqueueparameters.html#cfn-pipes-pipe-pipetargetsqsqueueparameters-messagededuplicationid
```
wrong link

WtfJoke replied:
good catch, fixed it

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,83 @@
+/**
+ * An EventBridge Pipes target that sends messages to a StepFunction.
+ */
+export class StepFunctionTarget implements ITarget {
```
```suggestion
export class StateMachineTarget implements ITarget {
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,83 @@
+ * InvocationType for invoking StepFunction.
+ * @see https://docs.aws.amazon.com/eventbridge/latest/pipes-reference/API_PipeTargetStateMachineParameters.html
+ */
+export enum StepFunctionInvocationType {
```
```suggestion
export enum StateMachineInvocationType {
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,83 @@
+    if (this.stateMachine instanceof StateMachine
+      && this.stateMachine.stateMachineType === StateMachineType.STANDARD
+      && this.invocationType === StepFunctionInvocationType.REQUEST_RESPONSE) {
+      throw new Error('Standard Workflows do not support REQUEST_RESPONSE invocation type');
```
Can you phrase this to include a solution for the user.
```suggestion
      throw new Error('STANDARD state machine workflows do not support the REQUEST_RESPONSE invocation type. Use FIRE_AND_FORGET instead.');
```

Also let's default to FIRE_AND_FORGET

WtfJoke replied:
adjusted to your suggestion and defaulted to `FIRE_AND_FORGET`

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,83 @@
+   *
+   * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetsqsqueueparameters.html#cfn-pipes-pipe-pipetargetsqsqueueparameters-messagededuplicationid
+   */
+  readonly invocationType: StepFunctionInvocationType;
```
I think we can default to FIRE_AND_FORGET here.

mrgrain reviewed (dismissed):
Looks great! Got a couple more minor renames and let's set a default for InvocationType so that most users won't have to specify parameters at all.

Also trying to get @RaphaelManke to review as well

RaphaelManke reviewed (commented):
Overall great job @WtfJoke. 
I have only one thing that I am not 100% happy about.
Can we define a common name for Step function that is used all over the place? 
Currently we have : 
- AWS Step Functions
- StepFunction
- StateMachine
- stepFunctionStateMachine
- sfn

The EventBridge targets constructs calls the target 
[SfnStateMachine](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_events_targets.SfnStateMachine.html)

Maybe we should align to the naming of that because its a name from an already stable package. 
Whats your thought in that @mrgrain ?

My suggestion is, name the target `SfnStateMachine`

WtfJoke commented:
> Can we define a common name for Step function that is used all over the place?


This is something I am not happy about it as well. 
I tried to Stick mostly to the term "Step Function", but the terms Step Function and State Machine are mostly used interchangeable. I'm open for either name.

Please keep in mind its not only the Targets name (StepFunctionTarket vs StateMachineTarget vs SfnStateMachineTarget (@RaphaelManke you meant to include `Target` right?)) but also the in the corresponding docs.

Currently in the docs:
> StateMachine StepFunction target properties.

or  

> InvocationType for invoking StepFunction.
> Invoke StepFunction asynchronously (`StartExecution`). See https://docs.aws.amazon.com/step-functions/latest/apireference/API_StartExecution.html for more details.

or in the README  
>### AWS Step Functions

I am not so eager on writing SfnStateMachine everywhere in the docs. I would favor choosing StepFunction or StateMachine there.
What do you think @RaphaelManke @mrgrain?

P.S: I've rebased my changes on latest main and incorporated the review feedback from @mrgrain.

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: daeeba805e8eba32bff81324aad65f814c0f81bc
* Result: FAILED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=cd8b2570-552a-49d7-8f3d-c8e8ad83d46a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 5a07a69a703c0da092a5a4d8c8176a6ef5f1e7f7
* Result: FAILED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=34291ede-9b6a-4d7f-8eda-7c8bab0e8d74%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

mrgrain commented:
> * AWS Step Functions

This should be used in docs, when referring to the service. 
"AWS Step Functions State Machine" can be used to refer to the resource in the docs, but when used multiple times "State Machine" is enough.

> * StepFunction

This should never be used, given that it is missing an `s` ;)

> * StateMachine

This the correct way to refer to just the resource.

> * stepFunctionStateMachine

Ignoring the case, this Would be correct way to name the combination of service & resource

> * sfn
> 
> The EventBridge targets constructs calls the target [SfnStateMachine](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_events_targets.SfnStateMachine.html)

I'm inclined to agree. `sfn` is the short version [used by the SDK](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/client/sfn/) as well.

> Maybe we should align to the naming of that because its a name from an already stable package. Whats your thought in that @mrgrain ?

Probably. I think we named the queue target wrong: `SqsTarget` when it really should be `SqsQueueTarget`.
But then all classes in the modules are targets, so maybe `SqsQueue` would be enough.

> My suggestion is, name the target `SfnStateMachine` 

I come to the same conclusion, in particular about dropping the `Target` suffix.
There could be an argument about calling it `StepFunctionsStateMachine` (I kind of like it better) but given the precedence of  EventBridge targets and that `sfn` is a semi official abbreviation, I'm okay with it.

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: f4b434bce336c4c10d2ccd7b5560409cbe8a2a64
* Result: FAILED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=a30b68bb-2f46-4924-bc93-256f30c312bf%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

mrgrain commented:
> This is something I am not happy about it as well. I tried to Stick mostly to the term "Step Function", but the terms Step Function and State Machine are mostly used interchangeable. I'm open for either name.

They should not be used interchangeably. We should make sure to explicitly call out the resource we are integrating with, which is the AWS Step Functions State Machine. Or shorter State Machine.

When grouping things, we tend to use the service. So that's why the file is named `stepfunctions.ts` or in the docs we might have a heading "AWS Step Functions". The idea being that if there were a second resource in the same service/module they would go into the same group. Of course in practice this rarely happens.

> Please keep in mind its not only the Targets name (StepFunctionTarket vs StateMachineTarget vs SfnStateMachineTarget (@RaphaelManke you meant to include `Target` right?)) but also the in the corresponding docs.

Yeah I think we should drop `Target` from the name. We can still refer to it in the docs as "`SfnStateMachine` target" when we want to make it explicit we are talking about a target.

> > InvocationType for invoking StepFunction.
> > Invoke StepFunction asynchronously (`StartExecution`). See https://docs.aws.amazon.com/step-functions/latest/apireference/API_StartExecution.html for more details.

Just because you called it out here, I think this is a nice example why it's important to differentiate between the service and the resource. StepFunction (without `s) doesn't exist. And StepFunctions (the service) doesn't have an InvocationType. Only the State Machine can be invoked with an InvocationType.

> I am not so eager on writing SfnStateMachine everywhere in the docs. I would favor choosing StepFunction or StateMachine there. What do you think @RaphaelManke @mrgrain?

I can go over the docs again to give some concrete notes. In example code it would be `SfnStateMachine`. In copy text, it would likely be "State Machine" or "the State Machine target" or "parameters of the State Machine target", etc.

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/README.md:
```diff
@@ -27,6 +27,11 @@ For more details see the service documentation:
+The following targets are supported:
+
+1. `targets.SqsTarget`: [Send event source to a Queue](#amazon-sqs)
+2. `targets.StateMachineTarget`: [Start a State Machine from an event source](#aws-step-functions)
```
Pending the code rename
```suggestion
2. `targets.SfnStateMachine`: [Invoke a State Machine from an event source](#aws-step-functions)
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/README.md:
````diff
@@ -63,3 +68,57 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
 });
 ```
+
+### AWS Step Functions
````
This makes sense as a heading. It could also be
```suggestion
### AWS Step Functions State Machine
```

WtfJoke commented:
I kinda prefer `StepFunctionsStateMachine` over `SfnStateMachine`, but in the end its up to you to decide.

> I can go over the docs again to give some concrete notes. In example code it would be SfnStateMachine. In copy text, it would likely be "State Machine" or "the State Machine target" or "parameters of the State Machine target", etc.

Once we set on a name, I can go through the code+docs again and afterwards it would be great to have a review again. IMHO no need for docs review in advance (but as you like).

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/README.md:
````diff
@@ -63,3 +68,57 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
+});
+```
+
+Specifying the Invocation Type of the State Machine.
````
This was already good. Slightly more explicit:
```suggestion
Specifying the Invocation Type when the target State Machine is invoked:
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/README.md:
````diff
@@ -63,3 +68,57 @@ const pipe = new pipes.Pipe(this, 'Pipe', {
+});
+```
+
+The target input can be transformed:
````
```suggestion
The input to the target State Machine can be transformed:
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,84 @@
+import { StateMachine, StateMachineType } from 'aws-cdk-lib/aws-stepfunctions';
+
+/**
+ * StateMachine StepFunction target properties.
```
```suggestion
 * Parameters for the SfnStateMachine target
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,84 @@
+  readonly inputTransformation?: IInputTransformation;
+
+  /**
+   * Specify whether to invoke the Step Functions state machine synchronously (`REQUEST_RESPONSE`) or asynchronously (`FIRE_AND_FORGET`).
```
```suggestion
   * Specify whether to invoke the State Machine synchronously (`REQUEST_RESPONSE`) or asynchronously (`FIRE_AND_FORGET`).
```

mrgrain commented on packages/@aws-cdk/aws-pipes-targets-alpha/lib/stepfunctions.ts:
```diff
@@ -0,0 +1,84 @@
+}
+
+/**
+ * InvocationType for invoking StepFunction.
```
```suggestion
 * InvocationType for invoking the State Machine.
```

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 71acc78b6b514019883ca381788fdc37c5088f43
* Result: FAILED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=bf53777b-f888-4b8d-be8e-84d26cc2ae5c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: fc6326557c2f31fbf05cf86e6d9c9f03f03d656d
* Result: FAILED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=6a10c362-4f5a-4d13-896d-35caefeaf79b%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 962eb74440fa2ed19680788dbd29ce8efe3b242f
* Result: SUCCEEDED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=ea96fd7a-70fb-4e05-bdba-1eaf499b3d60%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

aws-cdk-automation commented:
### AWS CodeBuild CI Report

* CodeBuild project: AutoBuildv2Project1C6BFA3F-wQm2hXv2jqQv
* Commit ID: 2712ba1255da525cf457f8915c680584ceb646fb
* Result: SUCCEEDED
* [Build Logs](https://2cl995f30a.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=79ad73a4-9a33-40de-9d91-b7630d07059a%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*


<!--
CREATED BY GITHUB-CODEBUILD-LOGS
-->

WtfJoke commented:
Thanks a bunch!

aws-cdk-automation commented:
Comments on closed issues and PRs are hard for our team to see. If you need help, please open a new issue that references this one.
